moorhuhn-service/src/main/java/com/moorhuhnservice/moorhuhnservice/BaseClasses/Question.java

```diff
@@ -0,0 +1,96 @@
+package com.moorhuhnservice.moorhuhnservice.BaseClasses;
+
+
+import javax.persistence.*;
+@Entity
+@Table(name = "questions")
+public class Question {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Long id;
+    @Column(nullable = false)
+    private String level;
+    @Column(nullable = false)
+    private String question;
+    @Column(nullable = false)
+    private String rightAnswer;
+    @Column(nullable = false)
+    private String wrongAnswerOne;
+    @Column(nullable = false)
+    private String wrongAnswerTwo;
+    @Column(nullable = false)
+    private String wrongAnswerThree;
+    @Column(nullable = false)
+    private String wrongAnswerFour;

```

Comment:
Yes, that's one approach.
An approach where you always have to assume that there are four wrong answers.
It would also be possible to simply store a `Set<String> wrongAnswers`, which changes little, but is far easier to use and modify.

Reply:
there are allways exactly 4 wrong answers, therefore this is perfectly fine in my opinion.